BDKSwiftExampleWallet/Extensions/BDK+Extensions/LocalOutput+Extensions.swift

```diff
@@ -6,17 +6,25 @@
 //
 
 import BitcoinDevKit
+import Foundation
 
 //#if DEBUG
 extension LocalOutput {
     static var mock = LocalOutput(
         outpoint: OutPoint(
-            txid: "txid",
+            txid: try! Txid.fromBytes(
```

Comment:
Txid changed from String to type https://github.com/bitcoindevkit/bdk-ffi/commit/23d14f05e07276e1c5bce47bd453d2a0f8b8763d

Would be nice to have a String constructor again somehow